.github/CODEOWNERS

```
@@ -103,6 +103,9 @@ nixos/modules/installer/tools/nix-fallback-paths.nix  @raitobezarius @ma27
 # NixOS QEMU virtualisation
 /nixos/virtualisation/qemu-vm.nix           @raitobezarius
 
+# ACME
+/nixos/modules/security/acme                @arianvp @flokli @m1cr0man @aanderse @emilazy
```

```suggestion
/nixos/modules/security/acme                @arianvp @flokli @aanderse # no merge permissions @m1cr0man @emilazy
```
since those entries won't work anyway, we might as well keep the error list above shorter

---

Ah that's disappointing. I really would like to get pings for acme PRs, but if GitHub permissions complicate it then so be it.

---

Agreed. there's an issue tracking this feature request here : https://github.com/NixOS/ofborg/issues/568